www/src/css/snap.css

```diff
@@ -33,7 +33,10 @@
 
 .b-snap__actions .b-installer {
   margin:0 10px;
-  width:auto;
+}
+
+.b-snap__actions .b-installer__button {
```

Comment:
I'll keep that in mind, cheers. 

The complication with this specific case is that the markup for the button is rendered using a shared partial and the different pages that use it call for buttons of different widths hence the my reliance on the class of a containing element.